client/src/components/Map.jsx

```diff
@@ -22,29 +22,24 @@ let config = {
     minZoom: 7,
     scrollwheel: false,
     infoControl: false,
-    attributionControl: false,
+    attributionControl: false
   },
   tileLayer: {
     uri: 'https://tiles.freemap.sk/T/{z}/{x}/{y}.png',
     params: {
       minZoom: 7,
       id: '',
-      accessToken: '',
-    },
-  },
+      accessToken: ''
+    }
+  }
 };
 
 class Map extends Component {
   constructor(props) {
     super(props);
     this.state = {
       map: null,
-      tileLayer: null,
```

Comment:
v riadku 168 mas setState s tileLayerom, nebude to robit problem?

Reply:
pozrem na to

Reply:
ten stav sa sice setuje ale nikde sa nevyuziva dam to prec aj z toho setovania

Reply:
OK